scripts/src/image-viewer.js

```diff
@@ -36,4 +39,14 @@ if (image_source) {
             full_screen_button.focus();
         }
     });
-}
\ No newline at end of file
+
+    canvas.addEventListener("focus", () => {
```

Comment:
Hi James - could you make a tiny tweak to this. Because it's possible that `document.getElementsByClassName` might  return an empty `HTMLCollection` could we wrap the calls to `addEventListener` in either a `try / catch` or an `if`?